utils/training_loop.py

```diff
@@ -1,270 +1,104 @@
-from utils.NN_utils import *
-from utils.airsim_manager import AirsimManager
-from utils.logger import Logger
-from utils.rl import RL
-
-
-def training_loop_ido(config):
-    # classes init
-    logger = Logger(config)
-    airsim = AirsimManager(config)
-    nn_handler = NN_handler(config)
-    rl = RL(config, logger, airsim, nn_handler)
+import matplotlib.pyplot as plt
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
+from gym_enviroment import AirSimGymEnv
+from airsim_manager import AirsimManager
+from stable_baselines3.common.logger import configure
+import pandas as pd
+from config import Config
+import random
+
+def training_loop_ariel_step(config, path):
+    all_rewards = []
+    new_logger = configure(path, ["stdout", "csv", "tensorboard"])
+    env = DummyVecEnv([lambda: AirSimGymEnv(config, AirsimManager(config))])
+    model = PPO('MlpPolicy', env, verbose=1, learning_rate=0.0001, n_steps=160, batch_size=160)
+    #model = PPO('MlpPolicy', env, verbose=1)
+    model.set_logger(new_logger)
+
+    def train_model():
+        total_timesteps = 160
+        print('Model learning')
+
+        model.learn(total_timesteps=total_timesteps)
+        print('Model learned ************')
+        env.envs[0].resume_simulation()
 
     if config.ONLY_INFERENCE:
-        rl.network = nn_handler.load_weights_to_network(rl.network)
+        model = PPO.load(config.LOAD_WEIGHT_DIRECTORY)
+        print("Loaded weights for inference.")
 
-    # Initialize counters for the experiment
     collision_counter = 0
     episode_counter = 0
     steps_counter = 0
+    total_steps = 0
 
-    # Main loop for each episode
     for episode in range(config.MAX_EPISODES):
-
-        # choose constant action for car2 for whole trajectory
-        if np.random.randint(2) == 1:
-            config.CAR2_CONSTANT_ACTION = 0
+        print(f"@ Episode {episode + 1} @")
+        car2_side = random.choice(["left", "right"])
```

Comment:
consider moving this section into a function in airsim manager, and also consider make hard coded values (90/[0,-30]) to config/class definition.